### ISSUE TITLE
Customize bubble size

### ISSUE THREAD

**Issue by nickyk2407** [igalata/Bubble-Picker#60]
@igalata 
Thanks for this library. It is very useful.
I have few queries:

1. Could help us to change the size of specific bubble before we draw?
2. Refresh the bubble view after select and deSelect call.
  For ex: change bubble text size or background color on bubble select or deSelect

Below is the code snippet I am using. This is changing selected bubble background color and text color but selected bubble size becomes smaller ( of the same size of non selected bubbles).

picker.setListener(new BubblePickerListener() {
            @Override
            public void onBubbleSelected(@NotNull PickerItem pickerItem) {
                pickerItem.setGradient(new BubbleGradient(selectedColors.getColor((1 * 2) % 2, 0),
                        selectedColors.getColor((1 * 2) % 2 + 1, 0), BubbleGradient.HORIZONTAL));
                pickerItem.setTextColor(ContextCompat.getColor(BubblePickerActivity.this, R.color.black));
                picker.onPause();
                picker.onResume();
            }

            @Override
            public void onBubbleDeselected(@NotNull PickerItem pickerItem) {

            }
        });

**Comment by lazy-coder-10** [comment]
Yes you can customize bubble size for each item with little modification on the library 

In Engine.Kt file inside the build function, after the for loop you can add your custom bubbleRadius as I am generating 40..100 random number for each items.

`for (i in 0..bodiesCount - 1) {
    val bubbleRadius = interpolate(0.1f, 0.25f, (40..100).random() / 100f)
    .......................
    ......................
 }`


Screenshot : 
![Screenshot_1615271291](https://user-images.githubusercontent.com/13958364/110428328-1727c500-80cf-11eb-8fc3-e8448ee1931f.png)
